recompose-idea/build.gradle.kts

```diff
@@ -24,28 +25,27 @@ dependencies {
 }
 
 intellij {
-    pluginName = "Recompose"
-    version = "2019.3"
-    updateSinceUntilBuild = false
-    downloadSources = true
-    updateSinceUntilBuild = true
-
-    setPlugins("IntelliLang", "Kotlin")
+    version.set("2022.1")
+    updateSinceUntilBuild.set(false)
+    updateSinceUntilBuild.set(false)
+    downloadSources.set(true)
+    plugins.set(listOf("IntelliLang", "Kotlin"))
 }
 
 tasks {
-    // Set the compatibility versions to 1.8
-    withType<JavaCompile> {
-        sourceCompatibility = "1.8"
-        targetCompatibility = "1.8"
-    }
-    listOf("compileKotlin", "compileTestKotlin").forEach {
-        getByName<KotlinCompile>(it) {
-            kotlinOptions.jvmTarget = "1.8"
+    withType<KotlinCompile> {
+        kotlinOptions {
+            jvmTarget = JavaVersion.VERSION_11.toString()
+            freeCompilerArgs = listOf("-progressive")
         }
     }
+
     patchPluginXml {
-        sinceBuild("193")
-        untilBuild("202.*")
+        sinceBuild.set("193")
+        untilBuild.set("202.*")
+    }
+    runPluginVerifier {
+        ideVersions.set(listOf("2022.1"))
```

Comment:
Why only use the latest version? If I would like to run it on older versions that could be compatible with the plugin why not add them? This could be directly run with Android Studio for example.

Reply:
I understand, but if you're aiming to use this plugin on Android Studio you need to support older versions because Android Studio does not use the latest version of the IDE. You have to take it into account.